### DISCUSSION TITLE
Some stable SS scores have slightly less score than the maximum achievable by the mod combination

### DISCUSSION THREAD

**Discussion by felipemarins** [ppy/osu#26967] [Q&A] [answered]
I don't know if it's something wrong with the conversion code or if it's the bug with max_combo reported on the osu-web channel on discord. Here are some scores where it occurs:

- https://osu.ppy.sh/scores/1727138922 (should be at least 1 020 000)
- https://osu.ppy.sh/scores/533589830 (should be at least 1 080 000)
- https://osu.ppy.sh/scores/942780793 (should be at least 1 020 000)

**Comment by sz-da** [comment, marked as answer]
In this case, it's because the mod multipliers are rounded in game so the value isn't exact. The exact value for HDCL is 1.0176 and HDHRCL is 1.078656, which matches the scores mentioned.

**Reply by Rekunan** [reply]
Also noting that the multipliers don't add up, but multiply. Perfect example is HDHR beating DT cause it's actually 1.06*1.06=1.124 and not 1.06+1.06=1.12.
